cvmfs/catalog_mgr_impl.h

```diff
@@ -1,3 +1,13 @@
+/*
+ * catalog_mgr_impl.h
+ *
+ *  Created on: Aug 13, 2015
+ *      Author: molina
+ */
+
+#ifndef CVMFS_CATALOG_MGR_IMPL_H_
+#define CVMFS_CATALOG_MGR_IMPL_H_
+
 /**
```

Comment:
Please move forward (see above).  Perhaps you got confused about my earlier comment that cpp fixes can be a separate pull request.  That refers to existing problems, i.e. a pull request can contain a number of fixes for the existing code base.  New PRs should be correct before merging.